sklearn/inspection/_partial_dependence.py

```diff
@@ -58,6 +58,10 @@ def _grid_from_X(X, percentiles, grid_resolution):
         The number of equally spaced points to be placed on the grid for each
         feature.
 
+    custom_values: dict
+        Mapping from column index of X to an array-like of values where
+        the partial dependence should be calculated for that feature
```

Comment:
nit: period here

Reply:
"Mapping from column index of X" isn't accurate, it's actually mapping the element number from the list of user-provided features. Not sure if this will have downstream effects.
i.e. If user specifies that they want PDP for feature 5 only, the key generated in the `partial_dependence` function will assign this feature as 0, not 5.

Reply:
I think the docstring is accurate. At this point, `X` has been subset to the features in the `features` array with `_safe_indexing` and the `custom_values` array is modified so that it maps the column index in the subset array to the array of values.